site/src/patterns/accordion.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Accordion
+description: Accordion patterns in the Atlas Design System
+template: standard
+---
+
+# Accordion patterns
+
+Because the `details` component has so many possible permutations, it can be helpful to provide examples of certain styles.
+
+## Accordion
+
+This accordion uses the `open` attribute to expand a designated panel on page load. Grouping `.accordion` components together using the `name` attribute, prevents more than 1 panel from being open at a time. Atomics classes are used to style the accordion `<details>` and `<summary>` elements.
+
+```html
+<details class="accordion background-color-body-accent border-radius margin-block-xxs">
+	<summary class="font-weight-semibold padding-xs">
+		<div class="accordion-header font-weight-semibold">Accordion headline 1 text</div>
+	</summary>
+	<div class="accordion-content padding-xs">
+		<p class="margin-bottom-xxs">
+			Viverra aliquet eget sit amet tellus cras. Nisl nisi scelerisque eu ultrices vitae.
+			Condimentum id venenatis a condimentum. Ut eu sem integer vitae.
+		</p>
+		<h4 class="font-weight-semibold">Accordion content subheading</h4>
+		<article class="card margin-top-xs">
+			<div class="card-template">
+				<p class="card-supertitle">Content Type</p>
+				<a href="#" class="card-title margin-bottom-none">The headline of the content item</a>
+			</div>
+		</article>
+	</div>
+</details>
+<details class="accordion background-color-body-accent border-radius margin-block-xxs">
+	<summary class="padding-xs">
+		<div class="accordion-header font-weight-semibold">Accordion headline 2 text</div>
+	</summary>
+	<div class="accordion-content padding-xs">
+		<p class="margin-bottom-xxs">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ullamcorper velit sed ullamcorper morbi tincidunt ornare massa
+			eget egestas.
+		</p>
+		<h4 class="font-weight-semibold">Accordion content subheading</h4>
+		<article class="card margin-block-xs">
+			<div class="card-template">
+				<p class="card-supertitle">Content Type</p>
+				<a href="#" class="card-title margin-bottom-none">The headline of the content item</a>
+			</div>
+		</article>
+		<article class="card">
+			<div class="card-template">
+				<p class="card-supertitle">Content Type</p>
+				<a href="#" class="card-title margin-bottom-none">The headline of the content item</a>
+			</div>
+		</article>
+	</div>
+</details>
+```
+
+```html
+<details class="accordion accordion-lg border-top">
+	<summary>
+		<div class="accordion-header margin-left-xxs">
+			<h3 class="font-weight-semibold font-size-h5 line-height-normal">Accordion header 1 text</h3>
+			<p class="font-size-md color-text-subtle">Accordion subheader text</p>
```

Comment:
Two things about this nesting:

`<summary>`, having an implicit `button` role, will likely obliterate any semantics inside of it, meaning this `h3` will semantically not be treated as an `h3`. It might be worth just switching this to something more generic like a `<p>` and still applying the larger font size and whatnot - that way, we keep the presentation, but don't imply things like impacting heading-based navigation.

Secondly, every element in here gets collapsed into forming the `summary` button's accessible name - so in this case, the `summary` is named "Accordion header 1 text Accordion subheader text." Screenreaders will announce that in full, no differentiation between children, and in real world use cases, this could get quite verbose (especially when you throw in translation). Even worse, _some_ speech-dictation users would need to say this whole name in full to trigger the accordion: "Click Accordion header 1 text Accordion subheader text." This can also get really tedious in real world use cases.

I say this not to say that we _can't_ do this kind of subheading design, but rather, to suggest we minimize how often we do it, since it can introduce some heavy tediousness for some disabled users.

Reply:
Understood, this pattern was included because it's being used on the [Career Path](https://learn.microsoft.com/en-us/training/career-paths/ai-engineer), if we want to discourage it we don't need to document it in atlas, was just documenting the existing patterns we'll be replacing with this component to deprecate the _details.scss in docs-ui.

Want me to remove this pattern from atlas? Updated it to use `<p>` tags

Reply:
Added MDN link to this pattern about the button role's effects on semantics on L63